pkg/config/admission/config.go

```diff
@@ -0,0 +1,43 @@
+package admission
+
+import (
+	"strings"
+
+	"github.com/horizoncd/horizon/pkg/admission/models"
+)
+
+type FailurePolicy string
+
+func (f FailurePolicy) Eq(other FailurePolicy) bool {
+	return strings.EqualFold(string(f), string(other))
+}
+
+const (
+	FailurePolicyIgnore FailurePolicy = "ignore"
+	FailurePolicyFail   FailurePolicy = "fail"
+)
+
+type ClientConfig struct {
+	URL      string `yaml:"url"`
+	CABundle string `yaml:"caBundle"`
+	Insecure bool   `yaml:"insecure"`
+}
+
+type Rule struct {
+	Resources  []string           `yaml:"resources"`
+	Operations []models.Operation `yaml:"operations"`
+	Versions   []string           `yaml:"versions"`
+}
+
+type Webhook struct {
+	Name           string        `yaml:"name"`
```

Comment:
Any considerations about using the field `Name`? It look likes it's not printed in any logs. I think that will be a good use case to debug if it's printed in logs.

Reply:
removed